api/apps/api/src/modules/access-control/access-control.module.ts

```diff
@@ -4,13 +4,14 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { UsersProjectsApiEntity } from '@marxan-api/modules/access-control/projects-acl/entity/users-projects.api.entity';
 import { UsersScenariosApiEntity } from '@marxan-api/modules/access-control/scenarios-acl/entity/users-scenarios.api.entity';
 
-import { ProjectAccessControl } from './projects-acl/project-access-control';
-import { ProjectAclService } from './projects-acl/project-acl.service';
-import { ProjectAclController } from './projects-acl/project-acl.controller';
-import { ProjectAclModule } from './projects-acl/project-acl.module';
-import { ScenarioAclModule } from './scenarios-acl/scenario-acl.module';
-import { ScenarioAccessControl } from './scenarios-acl/scenario-access-control';
-import { ScenarioAclService } from './scenarios-acl/scenario-acl.service';
+import { ProjectAccessControl } from '@marxan-api/modules/access-control/projects-acl/project-access-control';
+import { ProjectAclService } from '@marxan-api/modules/access-control/projects-acl/project-acl.service';
+import { ProjectAclController } from '@marxan-api/modules/access-control/projects-acl/project-acl.controller';
+import { ProjectAclModule } from '@marxan-api/modules/access-control/projects-acl/project-acl.module';
+import { ScenarioAclModule } from '@marxan-api/modules/access-control/scenarios-acl/scenario-acl.module';
+import { ScenarioAccessControl } from '@marxan-api/modules/access-control/scenarios-acl/scenario-access-control';
+import { ScenarioAclService } from '@marxan-api/modules/access-control/scenarios-acl/scenario-acl.service';
+import { ScenarioAclController } from '@marxan-api/modules/access-control/scenarios-acl/scenario-acl.controller';
```

Comment:
hmm, why? `project-acl` and `scenarios-`acl` are the submodules of `access-control` 🤔

Reply:
A couple of lines above it is using `@marxan-api` imports and I thought it should be used as well here.

However, giving it a deeper thought, I think the two lines above are the entitites and, at least one of them, used to be placed under the projects submodule before and maybe that's why it uses this.

Should I change all of them in this case to relative, right? (I'm asking but I will probably do it right away)

Reply:
@rubvalave you can also consider using index.ts files for each submodule as well to think through what exactly given module should expose and why